Feature/trap blog index

traplocz opened the pull request:
My first pull request

traplocz committed:
Blog and index html ready

traplocz committed:
Index and blog html ready

noelgarces commented on views/blog/blogs.ejs:
```diff
@@ -1,2 +1,46 @@
+</style>
+</head>
+<body>
+<header>
+    <h1>ifElse Blog</h1>
+    <i class="far fa-envelope"></i>
+</header>
+<h2>Web Development beginner students are encouraged to join a study group</h2>
+<h3 class="inlinefor-thedate-author">Sept 25, 2020 / by</h3><a href="#"> TrapLocz</a>
+    <br>
+<img height="500px" width="1200px" src="https://i.ibb.co/WpYJ5pT/Full-Stack-Developer-Road-Map-How-To-Become-A-Full-Stack-Developer-Edureka.png" alt="Image of blog post 1 - Full-Sack Web Development tools">
+<p>
+    I've always been a fan of study groups. Honestly, being apart of a small study group gives me the feel of a class setting with classmates. It helps me learn of different advanced course concepts and evaluating the topics I've learned. 
+    This study group filled the empty place of my search of communion within Web Development. Beginners of Html, Css, and Js such as myself connecting with more experienced people to work together on multiple things such as group projects, coding challenges, etc. If you enjoy studying alone, then that is your choice. If you are looking for a group to study with, I highly recommend joining one.
+</p>
+
+<h2>More blogs coming soon!</h2>
+<h3 class="inlinefor-thedate-author">Sept 26, 2020 / by</h3><a href="#"> IfElseCode</a>
+    <br>
+<img height="500px" width="1200px" src="https://i.ibb.co/M6fgzKV/static-dynamic2.jpg" alt="Image of blog post 2 - Example image">
+<p>Blog 2 text</p>
+
+<h2>More blogs coming soon!</h2>
+<h3 class="inlinefor-thedate-author">Sept 27, 2020 / by</h3><a href="#"> IfElseCode</a>
+<p>Blog 3 text</p>
```
- The lines that I am commenting on are the only ones you need. Please remove the rest of the boilerplate.

- As for the styles you are going to apply please make a folder inside inside /css called blog and add all your styles in their. 

- Then link that css file in the main-layout.css

traplocz replied:
Ok

traplocz committed:
Merge branch 'master' into feature/TrapBlogIndex

traplocz committed:
Updated pull request

Removed unessesary doctype html code

Made a folder called: \blog
nested in
public\css

Created 2 Css files called:
index.css
blog.css

Then linked those css file in the main-layout.css
located in: views\layouts\main-layout.ejs

traplocz commented:
new comment, testing..
